AlfrescoDocumentPickerFileProvider/AFPItemIdentifier.m

```diff
@@ -38,13 +38,20 @@ + (NSFileProviderItemIdentifier)itemIdentifierForSuffix:(NSString *)suffix andAc
 + (NSFileProviderItemIdentifier)itemIdentifierForSuffix:(NSString *)suffix andAccountIdentifier:(NSString *)accountIdentifier
 {
     NSString *identifier;
-    if(suffix)
+    if(accountIdentifier)
```

Comment:
I would recommend when matching strings, just as a precaution, to make the checks against the string's length --> "if (accountIdentifier.length) {...}". This would stand against the string being nil and also against the string being empty